### PR TITLE
Fix deploy warning

### PR DESCRIPTION
**Pull request by Simon-Busch**

### CODE REVIEW

**Comment by Simon-Busch** [comment]
This PR should fix the css warning and a couple of other smaller warnings 

I have checked in the site and didn't find bugs in the css but would love another set of eyes to make sure

**Comment by burgertime-c4** [comment]
I checked everything I could think of locally as well as in the deploy preview. Looks good to me! This is a big change though so we should babysit the site for a bit after merge/deploy

**Review by burgertime-c4 — APPROVED** [review]

**Review by joshuashort — APPROVED** [review]
ty! 🎉

**Pull request merged**
